### PR TITLE
land degradation layer

### PR DESCRIPTION
**Pull request by mluena**
### Overview

_Please write a description. If the PR is hard to understand, provide a quick explanation of the code._

### Designs

_Link to the related design prototypes (if applicable)_

### Testing instructions

_Please explain how to test the PR: ID of a dataset, steps to reach the feature, etc._

### Feature relevant tickets

_Link to the related task manager tickets_

---

## Checklist before submitting

- [ ] Meaningful commits and code rebased on `develop`.
- [ ] Update CHANGELOG
- [ ] If this PR adds feature that should be tested for regressions when
      deploying to staging/production, please add brief testing instructions
      to the deploy checklist

### CODE REVIEW

File: client/cypress/e2e/navigation.cy.ts [hunk 1]
```diff
@@ -4,8 +4,7 @@ describe('navigation', () => {
   });
 
   it('access to project detail page and switch dashboard', () => {
-    cy.visit('/');
-    cy.get('a[data-cy="project-item-link"]').first().click();
+    cy.visit('/projects/indonesia-landscape');
```
Comment (anamontiaga) [inline client/cypress/e2e/navigation.cy.ts:7]: This should not work as now we have project_code on the url. Should be something like:
`projects/AFoCO_036_2022`

File: client/src/containers/datasets/layers/projects/hooks.tsx [hunk 2]
```diff
@@ -34,11 +39,56 @@ export function useLayers({
...
+      maxzoom: 18,
+      minzoom: 6,
+    },
+    {
+      id: 'projects_fill',
+      type: 'fill',
+      source: 'projects',
+      'source-layer': 'afoco_locations_full',
+      paint: {
+        // 'fill-color': [
```
Comment (anamontiaga) [inline client/src/containers/datasets/layers/projects/hooks.tsx:78]: Do we need this comment code for the future or can we remove?

File: client/src/containers/datasets/layers/projects/legend.tsx [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
-  return <div className="h-3 w-3 rounded-full bg-green-500" />;
+import LegendSettings from '@/containers/legend-settings';
+import { LayerSettings } from '@/types/layers';
+
+export default function Legend({ settings }: { settings: LayerSettings }) {
+  return (
+    <div className="flex items-center space-x-2">
+      <div className="h-3 w-3 rounded-full bg-green-500" />
+      <p className="text-xs capitalize">{settings.id}</p>
+      <LegendSettings settings={settings} />
```
Comment (anamontiaga) [inline client/src/containers/datasets/layers/projects/legend.tsx:9]: Think we need a justify between here
Screenshot 2024-03-21 at 11 22 32

File: client/src/containers/legend-settings/index.tsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+import { LayerSettings } from '@/types/layers';
+
+import { useSyncLayers } from '@/hooks/datasets/sync-query';
+
+import { Popover, PopoverContent, PopoverTrigger } from '@/components/ui/popover';
+import { Slider } from '@/components/ui/slider';
+import { Tooltip, TooltipContent, TooltipTrigger } from '@/components/ui/tooltip';
+
+const LegendSettings = ({ settings }: { settings: LayerSettings }) => {
```
Comment (anamontiaga) [inline client/src/containers/legend-settings/index.tsx:15]: When select two layers i see no legends in map. 
Also Land degradation legend is without styles.
Screenshot 2024-03-21 at 11 24 00

File: cms/package.json [hunk 5]
```diff
@@ -7,7 +7,8 @@
     "dev": "strapi develop",
     "start": "strapi start",
     "build": "strapi build",
-    "strapi": "strapi"
+    "strapi": "strapi",
+    "types": "strapi ts:generate-types"
```
Comment (anamontiaga) [inline cms/package.json:11]: Does this prevent us from having to do the build inside the types folder?

File: client/src/containers/datasets/layers/projects/hooks.tsx (hunk 2 above)
Comment (mluena) [reply client/src/containers/datasets/layers/projects/hooks.tsx:78]: yes!

**Pull request merged**